sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/MapsGeofenceEnteredEventData.java

```diff
@@ -5,7 +5,37 @@
 package com.azure.messaging.eventgrid.systemevents;
 
 import com.azure.core.annotation.Immutable;
+import java.util.List;
 
-/** The MapsGeofenceEnteredEventData model. */
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Maps.GeofenceEntered event. */
 @Immutable
-public final class MapsGeofenceEnteredEventData extends MapsGeofenceEventProperties {}
+public final class MapsGeofenceEnteredEventData extends MapsGeofenceEventProperties {
```

Comment:
It's a bit odd that this file doesn't have syntax highlighting. Is this just a github issue or is there something odd about this java file?

Reply:
Syntax highlighting has no problem in my IDE. I guess it's a github issue